spacewalk/package/spacewalk.spec

```diff
@@ -109,20 +109,10 @@ Provides:       spacewalk-db-virtual = %{version}-%{release}
 Requires:       spacewalk-backend-sql-postgresql
 Requires:       spacewalk-java-postgresql
 Requires:       perl(DBD::Pg)
-%if 0%{?sle_version}
-%if 0%{?sle_version} >= 150400
-Requires:       postgresql14
-Requires:       postgresql14-contrib
-# we do not support postgresql versions > 14.x yet
-Conflicts:      postgresql-implementation >= 15
-Conflicts:      postgresql-contrib-implementation >= 15
-%else # not sle_version >= 150400
-Requires:       postgresql13
-Requires:       postgresql13-contrib
-# we do not support postgresql versions > 13.x yet
-Conflicts:      postgresql-implementation >= 14
-Conflicts:      postgresql-contrib-implementation >= 14
-%endif # if sle_version >= 150400
+%if 0%{?suse_version}
+# Actual version set by prjconf
+Requires:       postgresql-implementation = %{postgresql_version}
```

Comment:
Don't you want to have a default value for %{postgresql_version}` in the spec file in case the project config doesn't have one?

Reply:
Added, using 14 as default.